.circleci/config.yml

```diff
@@ -457,11 +476,8 @@ jobs:
       CMAKE_OPTIONS: -DSANITIZE=address
       MAKEFLAGS: -j 10
       CMAKE_BUILD_TYPE: Release
-    steps:
-      - checkout
-      - run: *run_build
-      - store_artifacts: *artifacts_solc
-      - persist_to_workspace: *artifacts_executables
+    <<: *steps_b_ubu
```

Comment:
You could use these steps for `b_ubu` as well. `store_artifacts: *artifacts_tools` can be done separately after these steps or you could just include it in `steps_b_ubu`. Storing built tools from sanitizer runs is not necessary but if it simplifies the config, I think it won't hurt.